osc_sdk_python/credentials.py

```diff
@@ -92,3 +98,9 @@ def get_profile(self):
 
     def get_url_extension(self):
         return 'hk' if 'cn' in self.region else 'com'
+
+    def get_email(self):
```

Comment:
As said before, I would prefer to remove the getter (in  another PR), than add other getter, that are not really useful